src/ColorContextMenu-Core.package/RadialMenuMorph.class/instance/outerFractionForItems.st

```diff
@@ -1,4 +1,4 @@
 accessing
 outerFractionForItems
 
-	^ self outerFraction - 0.05
\ No newline at end of file
+	^ self outerFraction "- 0.05"
```

Comment:
Hm ... I used to like the border. Why did you remove it?

Reply:
mostly because the border between top-level menu and submenu was twice as thick as the outer borders... also i just like the material look :)

Reply:
<img width="422" alt="Bildschirmfoto 2021-02-23 um 21 00 02" src="https://user-images.githubusercontent.com/37243299/108900616-1a747880-761a-11eb-9041-ce35c3678476.png">

Reply:
moved to #22